plugins/index.json

```diff
@@ -2,7 +2,7 @@
   {
     "name": "azure",
     "author": "Porter Authors",
-    "description": "Integrate Porter with Azure. Store Porter's data in Azure Cloud and secure your bundle's secrets in Azure Key Vault.",
+    "description": "Integrate Porter with Azure. Let you secure your bundle's secrets in Azure Key Vault.",
```

Comment:
nit: "Secure your bundle's secrets in Azure Key Vault" reads a little better IMO

Reply:
Agree, changed